docs/source/usage/workflows/addLaser.rst

```diff
@@ -7,18 +7,55 @@ Adding Laser
 
 There are several alternative ways of adding an incoming laser (or any source of electromagnetic field) to a PIConGPU simulation:
 
+#. selecting incident field profiles for respective boundaries in :ref:`incidentField.param <usage-params-core>`
 #. selecting a laser profile in :ref:`laser.param <usage-params-core>`
-#. selecting non-none incident field profiles for respective boundaries in :ref:`incidentField.param <usage-params-core>`
 #. using field or current background in :ref:`fieldBackground.param <usage-params-core>`
 
 These ways operate independently of one another, each has its features and limitations.
+Incident field- and laser profiles currently match one another.
+
+Incident Field
+""""""""""""""
+
+Incident field is an external field source producing a wave propagating inwards the simulation volume.
+The source is applied at a boundary of an axis-aligned box located inside the simulation area.
+The implementation is based on the total field/scattered field formulation described in detail :ref:`here <model-TFSF>`.
+A user sets offsets of each side of this box from global domain boundary in :ref:`incidentField.param <usage-params-core>`.
```

Comment:
offset of each side -> offsets from each side?

Reply:
No, these are offsets of sides of Huygens box from global domain box.
However, I removed `each side of` to make it more readable.